flow-typed/Claim.js

```diff
@@ -191,32 +191,3 @@ declare type ChannelPublishParams = {
   tags?: Array<string>,
```

Comment:
Unrelated, but maybe handle it as well if code still exists?

<img width="507" alt="image" src="https://user-images.githubusercontent.com/64950861/189362106-90e2e75c-6b2e-4afe-8b91-71eed5b8738b.png">